app/services/npq/create_participant_outcome.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+module NPQ
+  class CreateParticipantOutcome
+    include ActiveModel::Model
+    include ActiveModel::Attributes
+
+    attribute :cpd_lead_provider
+    attribute :participant_external_id
+    attribute :course_identifier
+    attribute :state
+    attribute :completion_date, :date
+
+    validates :cpd_lead_provider, :participant_external_id, presence: true
+    validates :completion_date, future_date: true, presence: { message: I18n.t(:missing_completion_date) }
+    validates :course_identifier, course: true, presence: { message: I18n.t(:missing_course_identifier) }
+    validates :state,
+              presence: { message: I18n.t(:missing_state) },
+              inclusion: {
+                in: ParticipantOutcome::NPQ::VALID_STATES.map(&:to_s),
```

Comment:
when validating states, could we only allow passed/failed as we don't want to allow providers to void outcomes. We will internally void outcomes ourselves if a declaration is voided or marked as ineligible